### DISCUSSION TITLE
Row update issue [Apr 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1720] [Discord Threads]
> 🙋‍♂️ **Morza (Chris)**  
## Hey guys, for some reason the row is not updating when I'm updating the data in a column. Any idea of what this could be? I'm using graphql with a non-external db  
[Screen_Recording_2022-04-08_at_12.25.59_PM.mov](https://cdn.discordapp.com/attachments/661905455894888493/961935205147504712/Screen_Recording_2022-04-08_at_12.25.59_PM.mov)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### do you see any issues in network logs ? Thank you for the details.    
####    
#### And is this happening for other tables as well ?  
#  
> 🙋‍♂️ **Morza (Chris)**  
#### I'm not even seeing the request being logged in the logs, and yea it's happening both for rest and graphql  
#  
> 💁 **o1lab**  
#### have DMed  
#  
> 💁 **o1lab**  
#### thanks for the details & debuggin Chris   
#### looks like this has to do with grammarly plugin on firefox browser   
#### we will see what could be done  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/961935207148163133)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
